log/cmt.go

```diff
@@ -0,0 +1,55 @@
+package log
+
+import (
+	cmtlog "github.com/cometbft/cometbft/libs/log"
+	"github.com/rs/zerolog"
+)
+
+var _ cmtlog.Logger = (*CometZerologWrapper)(nil)
```

Comment:
nit: i think this should live next to where its passed to comet instead of here. Log package shouldnt have comet as a dep.

Reply:
Ughhh I hate to agree here. 

Ideally, the build-time assertion should reside where the implementation resides (i.e. here), but being that we do not want to import the `CometBFT` lib, I agree we should move the build-time assertion to `server` where we pass it to `CometBFT`.